planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
thanks @MicheleGuerriero. I assume you are aware of #252 and the long open discussion on the DAM and then the DamGenerator, yes?
I cannot see `monitor-dam-generator` used in #252. Please coordinate among yourselves (@MicheleGuerriero, @PaoloCifariello and @kiuby88 ) and when you have a clear understanding of the situation re-open those conflicting PRs. Thanks